### DISCUSSION TITLE
Are you open for design contributions?

### DISCUSSION THREAD

**Discussion by fivenp** [balzack/databag#7] [Ideas]
Hey there! 

I discovered databag via Reddit and thought it might benefit here and there potentially from some design driven input. :)  

Therefor I've built https://contribute.design which helps designers and open source software to understand each other better. Basically I'm trying to help project get the guidelines for designers set up and create some visibility across designers for the project. 

My question now is – would you be open to participate in this initiative and can I potentially support you in any way here? :)

**Comment by balzack** [comment]
Absolutely! I have very little experience with design and would benefit immensely. I don't have any design resources at the moment, no style guide, official logo, or figma mock. Please let me know what I can do to attract interest in collaboration. I've taken the steps called out on your site, and added the badge to the top of the Readme. Thanks for building the site and contacting me.

**Comment by fivenp** [comment]
Amazing! Glad to hear that... I'm pretty sure there'll be a designer who'll stumble upon databag ready to help and improve things together with you :)
